src/applications/vaos/tests/e2e/community-care.cypress.spec.js

```diff
@@ -1,16 +1,16 @@
-import { initCommunityCareMock } from './vaos-cypress-helpers';
+import {
+  initCommunityCareMock,
+  mockFeatureToggles,
+} from './vaos-cypress-helpers';
 import moment from 'moment';
 
 describe('VAOS community care flow', () => {
-  beforeEach(() => {
```

Comment:
Removed the beforeEach because we need to call the init functions slightly differently between the two tests